tools/system_libs.py

```diff
@@ -1362,6 +1380,9 @@ def add_library(lib):
       if d not in shared.Settings.EXPORTED_FUNCTIONS:
         shared.Settings.EXPORTED_FUNCTIONS.append(d)
 
+  if shared.Settings.STANDALONE_WASM and '_main' in shared.Settings.EXPORTED_FUNCTIONS:
```

Comment:
could this be next to the other line talking about `STANDALONE_WASM`? or is there a reason it has to be here specifically?

Reply:
I somehow feel like crt1 should come first on the link line.. but maybe I'm being irrational :)